DependencyInjection/SonataUserExtension.php

```diff
@@ -356,6 +357,32 @@ public function configureMenu(array $config, ContainerBuilder $container)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function prepend(ContainerBuilder $container)
+    {
+        $bundles = $container->getParameter('kernel.bundles');
+
+        // Blacklist all API controllers when using JMSDiExtraBundle without NelmioApiDocBundle or FOSRestBundle. API
+        // controllers use annotations and would be warmed, throwing an exception.
+        if (isset($bundles['JMSDiExtraBundle']) &&
+            (!isset($bundles['NelmioApiDocBundle']) || !isset($bundles['FOSRestBundle']))) {
```

Comment:
I think this is equivalent to `!isset($bundles['NelmioApiDocBundle'], $bundles['FOSRestBundle'])`